Sat Apr 10 23:58:04 CEST 2021 : Servidor de red Apache Derby: Se ha iniciado 10.12.1.1 - (1704137) y está listo para aceptar las conexiones en el puerto 1527
----------------------------------------------------------------
Sat Apr 10 23:58:27 CEST 2021:
 Iniciando instancia a816c00e-0178-bdca-bfbc-ffffd84318d6 de Derby, versión The Apache Software Foundation - Apache Derby - 10.12.1.1 - (1704137): 
en el directorio de base de datos /Users/ruben/NetBeansProjects/Derby/fasa con el cargador de clases sun.misc.Launcher$AppClassLoader@66d3c617
Cargado desde file:/Users/ruben/NetBeansProjects/Derby/db-derby-10.12.1.1-bin/lib/derby.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_111-b14
user.dir=/Users/ruben/NetBeansProjects/Derby/db-derby-10.12.1.1-bin
os.name=Mac OS X
os.arch=x86_64
os.version=10.15.7
derby.system.home=/Users/ruben/NetBeansProjects/Derby
Cargador de clases de base de datos iniciado - derby.database.classpath=''
----------------------------------------------------------------
Sat Apr 10 23:58:27 CEST 2021:
Cerrando la instancia a816c00e-0178-bdca-bfbc-ffffd84318d6 en el directorio de base de datos /Users/ruben/NetBeansProjects/Derby/fasa con cargador de clases sun.misc.Launcher$AppClassLoader@66d3c617
Sat Apr 10 23:58:27 CEST 2021 Thread[DRDAConnThread_3,5,main] (DATABASE = fasa), (DRDAID = {2}), Cierre de la base de datos 'fasa'.
----------------------------------------------------------------
Sat Apr 10 23:58:30 CEST 2021:
 Iniciando instancia b6b2c0ae-0178-bdca-bfbc-ffffd84318d6 de Derby, versión The Apache Software Foundation - Apache Derby - 10.12.1.1 - (1704137): 
en el directorio de base de datos /Users/ruben/NetBeansProjects/Derby/fasa con el cargador de clases sun.misc.Launcher$AppClassLoader@66d3c617
Cargado desde file:/Users/ruben/NetBeansProjects/Derby/db-derby-10.12.1.1-bin/lib/derby.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_111-b14
user.dir=/Users/ruben/NetBeansProjects/Derby/db-derby-10.12.1.1-bin
os.name=Mac OS X
os.arch=x86_64
os.version=10.15.7
derby.system.home=/Users/ruben/NetBeansProjects/Derby
Cargador de clases de base de datos iniciado - derby.database.classpath=''
----------------------------------------------------------------
Sun Apr 11 00:05:09 CEST 2021:
 Iniciando instancia f33d40c7-0178-bdca-bfbc-ffffd84318d6 de Derby, versión The Apache Software Foundation - Apache Derby - 10.12.1.1 - (1704137): 
en el directorio de base de datos /Users/ruben/NetBeansProjects/Derby/e con el cargador de clases sun.misc.Launcher$AppClassLoader@66d3c617
Cargado desde file:/Users/ruben/NetBeansProjects/Derby/db-derby-10.12.1.1-bin/lib/derby.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_111-b14
user.dir=/Users/ruben/NetBeansProjects/Derby/db-derby-10.12.1.1-bin
os.name=Mac OS X
os.arch=x86_64
os.version=10.15.7
derby.system.home=/Users/ruben/NetBeansProjects/Derby
Cargador de clases de base de datos iniciado - derby.database.classpath=''
----------------------------------------------------------------
Sun Apr 11 00:05:09 CEST 2021:
Cerrando la instancia f33d40c7-0178-bdca-bfbc-ffffd84318d6 en el directorio de base de datos /Users/ruben/NetBeansProjects/Derby/e con cargador de clases sun.misc.Launcher$AppClassLoader@66d3c617
Sun Apr 11 00:05:09 CEST 2021 Thread[DRDAConnThread_3,5,main] (DATABASE = e), (DRDAID = {5}), Cierre de la base de datos 'e'.
Sun Apr 11 00:06:30 CEST 2021 Thread[DRDAConnThread_3,5,main] Cleanup action starting
java.sql.SQLException: No se ha encontrado la base de datos 'e'.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleDBNotFound(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(Unknown Source)
	at org.apache.derby.impl.drda.Database.makeConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.run(Unknown Source)
Caused by: ERROR XJ004: No se ha encontrado la base de datos 'e'.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(Unknown Source)
	... 21 more
============= begin nested exception, level (1) ===========
ERROR XJ004: No se ha encontrado la base de datos 'e'.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleDBNotFound(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(Unknown Source)
	at org.apache.derby.impl.drda.Database.makeConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.run(Unknown Source)
============= end nested exception, level (1) ===========
Cleanup action completed
Sun Apr 11 00:06:30 CEST 2021 Thread[DRDAConnThread_3,5,main] (DATABASE = e), (DRDAID = {6}), No se ha encontrado la base de datos 'e'.
----------------------------------------------------------------
Sun Apr 11 00:06:44 CEST 2021:
 Iniciando instancia 00df816d-0178-bdca-bfbc-ffffd84318d6 de Derby, versión The Apache Software Foundation - Apache Derby - 10.12.1.1 - (1704137): 
en el directorio de base de datos /Users/ruben/NetBeansProjects/Derby/sampleTAW con el cargador de clases sun.misc.Launcher$AppClassLoader@66d3c617
Cargado desde file:/Users/ruben/NetBeansProjects/Derby/db-derby-10.12.1.1-bin/lib/derby.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_111-b14
user.dir=/Users/ruben/NetBeansProjects/Derby/db-derby-10.12.1.1-bin
os.name=Mac OS X
os.arch=x86_64
os.version=10.15.7
derby.system.home=/Users/ruben/NetBeansProjects/Derby
Cargador de clases de base de datos iniciado - derby.database.classpath=''
----------------------------------------------------------------
Sun Apr 11 00:06:45 CEST 2021:
Cerrando la instancia 00df816d-0178-bdca-bfbc-ffffd84318d6 en el directorio de base de datos /Users/ruben/NetBeansProjects/Derby/sampleTAW con cargador de clases sun.misc.Launcher$AppClassLoader@66d3c617
Sun Apr 11 00:06:45 CEST 2021 Thread[DRDAConnThread_4,5,main] (DATABASE = sampleTAW), (DRDAID = {8}), Cierre de la base de datos 'sampleTAW'.
----------------------------------------------------------------
Sun Apr 11 00:06:47 CEST 2021:
 Iniciando instancia 4653820d-0178-bdca-bfbc-ffffd84318d6 de Derby, versión The Apache Software Foundation - Apache Derby - 10.12.1.1 - (1704137): 
en el directorio de base de datos /Users/ruben/NetBeansProjects/Derby/sampleTAW con el cargador de clases sun.misc.Launcher$AppClassLoader@66d3c617
Cargado desde file:/Users/ruben/NetBeansProjects/Derby/db-derby-10.12.1.1-bin/lib/derby.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_111-b14
user.dir=/Users/ruben/NetBeansProjects/Derby/db-derby-10.12.1.1-bin
os.name=Mac OS X
os.arch=x86_64
os.version=10.15.7
derby.system.home=/Users/ruben/NetBeansProjects/Derby
Cargador de clases de base de datos iniciado - derby.database.classpath=''
----------------------------------------------------------------
Mon Apr 12 23:33:42 CEST 2021: cerrando motor de Derby
----------------------------------------------------------------
Mon Apr 12 23:33:42 CEST 2021:
Cerrando la instancia 4653820d-0178-bdca-bfbc-ffffd84318d6 en el directorio de base de datos /Users/ruben/NetBeansProjects/Derby/sampleTAW con cargador de clases sun.misc.Launcher$AppClassLoader@66d3c617
